### ISSUE TITLE
Provide support  for v21 SCTP offer spec

### ISSUE THREAD

**Issue by dlukyanyuk** [keroserene/go-webrtc#107]
According https://bugs.chromium.org/p/webrtc/issues/detail?id=9484 new versions of Chrome are sending new offer format and answer cannot be generated.

For example, for RemoteDescription with info
`m=application 54111 UDP/DTLS/SCTP webrtc-datachannel
a=sctp-port:5000` 
pc.CreateAnswer does not produce any result - no error, no answer

Similiar issue for node implementation - https://github.com/node-webrtc/node-webrtc/issues/483

**Comment by uumaro** [comment]
Do you know, is this just a matter of using a newer version of Chromium to base libwebrtc off of?

**Comment by dlukyanyuk** [comment]
This occurs when one client uses go-webrtc library and another client uses browser (or mobile app) with newer version of Chromium.

**Comment by arlolra** [comment]
> Do you know, is this just a matter of using a newer version of Chromium to base libwebrtc off of?

Yeah, looks like node-webrtc fixed this by bumping to `branch-heads/70`
